.github/workflows/mobile-tests.yml

```
@@ -0,0 +1,47 @@
+        run: |
+          flutter test --coverage --file-reporter="json:coverage/app-results.json" --test-randomize-ordering-seed random
+
+      - name: Test Report
+        uses: dorny/test-reporter@v2
+        if: always()
+        with:
+          name: Flutter Tests
+          path: 'coverage/*.json'
+          reporter: flutter-json
+          fail-on-error: true
+        continue-on-error: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)